releasenotes/notes/add-kwargs-to-standard-gates-d3ebf755e1d30efd.yaml

```diff
@@ -0,0 +1,4 @@
+fixes:
+  - |
+    Added kwargs to standard gates for duration and unit.
```

Comment:
Thanks for being through and adding a release note, but we actually don't need one for this PR. We already documented this fix in https://github.com/Qiskit/qiskit/blob/main/releasenotes/notes/singletons-83782de8bd062cbc.yaml#L118 which hasn't been included in a release yet. This PR is fixing the fix to actually be complete (as opposed to a subset of gate classes) :)